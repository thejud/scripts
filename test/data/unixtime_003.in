this is a filter test
foo 1601021438 bar
something else
1601021467 end
start 1601021591594
1601021467
0001 no-op
9999999999999999999999 no-op
