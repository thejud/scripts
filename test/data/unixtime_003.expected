this is a filter test
foo 2020-09-25 08:10:38 UTC bar
something else
2020-09-25 08:11:07 UTC end
start 2020-09-25 08:13:11 UTC
2020-09-25 08:11:07 UTC
0001 no-op
9999999999999999999999 no-op
